drakrun/drakrun/main.py

```diff
@@ -203,9 +203,8 @@ def send_raw_analysis(self, sample, outdir: str, metadata, quality: str) -> None
         """
 
         headers = {
-            "type": "analysis-raw",
-            "kind": "drakrun-internal",
-            "quality": quality,
```

Comment:
quality intentionally removed?

Reply:
In latest versions it's persistent header, no need to pass it manually